src/components/community-link/community-link.js

```diff
@@ -0,0 +1,52 @@
+import '@brightspace-ui/core/components/link/link.js';
+
+import { getDocumentLocaleSettings } from '@brightspace-ui/intl/lib/common.js';
+
+const Link = customElements.get('d2l-link');
+
+class CommunityLink extends Link {
+
+	static get properties() {
+		return {
+			...Link.properties,
+			articleMap: { attribute: 'article-map', type: Object }
+		};
+	}
+
+	get link() {
+		const { language } = getDocumentLocaleSettings();
+		const { articleCode, langCode } = this._getArticleCode(language);
+
+		return `https://community.d2l.com/brightspace${langCode === 'en' ? '' : `-${langCode}`}/kb/articles/${articleCode}`;
+	}
+
+	render() {
+		this.href = this.link;
+		return super.render();
+	}
```

Comment:
I'd suggest something like the following. Otherwise when the language on the page changes you won't be notified.

```suggestion
	constructor() {
		super();
		this._localeSettings = getDocumentLocaleSettings();
	}
	connectedCallback() {
		super.connectedCallback();
		this._localeSettings.addChangeListener(this._onLangChange);
	}
	disconnectedCallback() {
		super.disconnectedCallback();
		this._localeSettings.removeChangeListener(this._onLangChange);
	}
	render() {
		const { articleCode, langCode } = this._getArticleCode(this._localeSettings.language);
		const href = `https://community.d2l.com/brightspace${langCode === 'en' ? '' : `-${langCode}`}/kb/articles/${articleCode}`;
		return html`<a href="${href}" class="d2l-link"><slot></slot></a>`;
	}
	_onLangChange() {
		this.requestUpdate();
	}
```